src/zeit/retresco/json/update.py

```diff
@@ -0,0 +1,38 @@
+from zeit.retresco.update import index_async
+import json
+import logging
+import zeit.cms.browser.view
+
+log = logging.getLogger(__name__)
+
+
+class UpdateKeywords(zeit.cms.browser.view.Base):
+
+    def __call__(self):
+        if (self.request.method == 'POST' and
+                not self.request.form.get('_body_decoded')):
```

Comment:
Das mit dem _body_decoded war in zeit.edit interessant, weil da evtl mehrfach auf den Body zugegriffen wurde. Hier brauchen wir das denke ich nicht.